src/main/java/edu/cornell/kfs/fp/businessobject/ScheduledAccountingLineUtil.java

```diff
@@ -0,0 +1,114 @@
+package edu.cornell.kfs.fp.businessobject;
+
+import static org.kuali.kfs.sys.document.validation.impl.AccountingDocumentRuleBaseConstants.ERROR_PATH.DOCUMENT_ERROR_PREFIX;
+
+import java.sql.Date;
+import java.util.Calendar;
+import java.util.TreeMap;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.businessobject.AccountingLine;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.util.GlobalVariables;
+
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.CUKFSKeyConstants;
+
+public class ScheduledAccountingLineUtil {
+	public static void copyFrom(ScheduledSourceAccountingLine toLine, AccountingLine other) {
```

Comment:
Well, I do value consistency, but I think PreEncumbranceAccountingLineUtil isn't consistent with other areas of the code. This method is only called once, so I don't see any value in having it in the util class, it only makes a developer spend more time looking to figure out what is going on. Does that make sense?